frontend/src/pages/Chat.jsx

```
@@ -374,6 +377,26 @@
 
               {/* Message Input */}
               <div className="p-4 bg-white border-t border-gray-100 flex items-center">
+                <input type="file"
+                ref={inputRef}
+                className="hidden"
+                onChange={(e)=> setFile(e.target.files[0])}
+                />
+                <button onClick={()=>inputRef.current.click()} className="p-2 rounded-full bg-gray-100 hover:bg-gray-200" title="Attach File">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)